pkg/storaged/client.js

```diff
@@ -1225,7 +1233,7 @@ function stratis2_fixup_pool_notifications(data) {
                     const props = data[path][iface];
                     if (props && props.Name) {
                         // The pool at 'path' got renamed.
-                        fixup_data[path] = { "org.storage.stratis2.pool.r1": { Name: props.Name } };
+                        fixup_data[path] = { "org.storage.stratis2.pool.r3": { Name: props.Name } };
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230530-123245-679f24f8-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)